release-notes/opensearch-security-dashboards-plugin.release-notes-1.3.11.0.md

```diff
@@ -0,0 +1,7 @@
+## 2023-05-29 Version 1.3.11.0
```

Comment:
The date should be updated to 1.3.11 release date

Reply:
Yeah I just missed the month :/